res/css/views/dialogs/security/_AccessSecretStorageDialog.pcss

```diff
@@ -117,6 +117,7 @@ limitations under the License.
                 .mx_AccessSecretStorageDialog_reset {
                     position: relative;
                     padding-inline-start: $spacingStart;
+                    font-weight: inherit;
```

Comment:
Why is this needed? What happens without it?

Reply:
The text goes bold, and in this location it makes no sense as we have iconography to back up the colour emphasis. Other sites use merely colour which is not sufficient for visual a11y

Reply:
ok, it might be nice to include a comment to say that?